roaring.go

```diff
@@ -1878,6 +1878,185 @@ func (rb *Bitmap) CloneCopyOnWriteContainers() {
 	rb.highlowcontainer.cloneCopyOnWriteContainers()
 }
 
+func (rb *Bitmap) NextValue(target int) int {
```

Comment:
These new 4 functions should be documented, something like...

```
// NextValue returns the next largest value in the bitmap, or -1
// if none is present. This function should not be used inside
// a performance-sensitive loop: prefer iterators if 
// performance is a concern.
```

Reply:
done.

Reply:
@bearrito I don't see a new commit, did you push your changes?

Reply:
pushed. mea culpa.